frontend/packages/container-security/src/components/image-manifest-vuln.tsx

```diff
@@ -302,28 +302,34 @@ export const ImageManifestVulnList: React.FC<ImageManifestVulnListProps> = (prop
 };
 
 export const ImageManifestVulnPage: React.FC<ImageManifestVulnPageProps> = (props) => {
+  const { showTitle = true, hideNameLabelFilters = true } = props;
+  const namespace = props.namespace || props.match.params.ns || props.match.params.name;
```

Comment:
@vikram-raj Why are we looking for `props.match.params.name` here?

Reply:
In the project details page of admin perspective, we get the namespace `props.match.params.name`, in dev perspective, we get the namespace in `props.match.params.ns` whereas in list page we get the namespace in `props.namespace`.